omod-common/src/test/java/org/mockingbird/test/rest/resource/BirdResource_1_9.java

```diff
@@ -60,6 +61,21 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 		return null;
 	}
 	
+	@Override
+	public Model getGETModel(Representation representation) {
+		return null;
+	}
+	
```

Comment:
I see a number of these methods simply return null. Shouldn't it be enough not to have them and assume that whichever class does not have them, null it is? Am looking at reducing the amount of code that we have to maintain.